libs/vm/vm.py

```
@@ -87,6 +87,29 @@ def set_interface_state(self, network_name: str, state: str) -> None:
         }
         ResourceEditor(patches=patches).update()
 
+    @classmethod
+    def from_existing(
+        cls,
+        name: str,
+        namespace: str,
+        client: DynamicClient,
+        os_distribution: str,
+    ) -> BaseVirtualMachine:
+        obj = cls.__new__(cls)  # noqa: FCN001
```

Ruff (0.14.10)

Unused `noqa` directive (unknown: `FCN001`)

Remove unused `noqa` directive

(RUF100)